core/hash/fetch_values_spec.rb

```diff
@@ -15,15 +16,7 @@
     end
 
     describe "with unmatched keys" do
-      it "raises a KeyError" do
-        ->{ @hash.fetch_values :z }.should raise_error(KeyError)
-        ->{ @hash.fetch_values :a, :z }.should raise_error(KeyError)
-      end
-
-      it "returns the default value from block" do
-        @hash.fetch_values(:z) { |key| "`#{key}' is not found" }.should == ["`z' is not found"]
-        @hash.fetch_values(:a, :z) { |key| "`#{key}' is not found" }.should == [1, "`z' is not found"]
-      end
```

Comment:
It's the group `describe "with unmatched keys" do` below which should be replace by `it_behaves_like`.